.github/workflows/pipeline.yaml

```diff
@@ -40,27 +38,9 @@ jobs:
       - name: Checkout files 
         uses: actions/checkout@v2
       - name: Build docker
-        run: docker build --target production --tag $DOCKER_HUB_USER/todo-app:prod-${{ github.sha }} .
+        run: docker build --target production --tag $DOCKER_HUB_USER/todo-app:prod-latest .
       - name: Push to docker hub
-        run: docker push $DOCKER_HUB_USER/todo-app:prod-${{ github.sha }}
-      - name: Heroku login credentials
-        run: |
-          cat > ~/.netrc <<EOF
-            machine api.heroku.com
-              login $HEROKU_EMAIL
-              password $HEROKU_API_KEY
-            machine git.heroku.com
-              login $HEROKU_EMAIL
-              password $HEROKU_API_KEY
-          EOF
-      - name: Add Heroku remote
-        run: heroku git:remote --app $HEROKU_APP_NAME
-      - name: Login into Heroku's registry
-        run: heroku container:login
-      - name: Tag for heroku
-        run: docker tag $DOCKER_HUB_USER/todo-app:prod-${{ github.sha }} registry.heroku.com/$HEROKU_APP_NAME/web
-      - name: Push to heroku
-        run: docker push registry.heroku.com/$HEROKU_APP_NAME/web
-      - name: Release heroku
-        run: heroku container:release web
+        run: docker push $DOCKER_HUB_USER/todo-app:prod-latest
+      - name: Re-pull and restart the app in azure
+        run: curl -dH -X POST 'https://\$mh-todo-app-2022:$AZURE_PASSWORD@mh-todo-app-2022.scm.azurewebsites.net/api/registry/webhook'
```

Comment:
Nice work - it's often easier to pull out the entire webhook but you've captured the sensitive bit so this is totally valid!

One minor suggestion is to add the `--fail` flag to this curl command; without it curl considers the call a success as long as it gets a response, even if the response is to say an error occurred!